app/models/article.rb

```diff
@@ -466,4 +466,18 @@ def self.time_delta(year = nil, month = nil, day = nil)
     to = to - 1 # pull off 1 second so we don't overlap onto the next day
     return from..to
   end
+
+  def merge_with(other_article_id)
+    # find each of the comments associated w/ the article
+    Article.find(other_article_id).comments.each do |comment|
```

Comment:
I tried this: self.comments += other_article.comments and then saving self.comments, but found that the other_article comments were not actually being saved. Is there another way to reassign the comments without iterating through each one?